app/helpers/application_helper.rb

```diff
@@ -1,4 +1,14 @@
 # frozen_string_literal: true
 
 module ApplicationHelper
+  def bootstrap_class_for(flash_type)
+    case flash_type
+    when 'success' then 'alert-success'
+    when 'error' then 'alert-danger'
+    when 'alert' then 'alert-warning'
+    when 'notice' then 'alert-info'
+    else
+      flash_type.to_s
```

Comment:
I think the 'else' condition should be set to a default value like 'alert-info'.  Now if you run the helper  bootstrap_class_for('dupa_blada') you won't format the bootstrap alert correctly.

BTW: I wonder why do you convert  flash_type to string``` flash_type.to_s ```. If flash_type is a symbol than the 'when' conditions are not correct.
``` when :success then 'alert-success' ```